roxie/ccd/ccdmain.cpp

```diff
@@ -1415,7 +1415,8 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
                     roxieServer->start();
                 }
 
-                queryFileCache().loadSavedOsCacheInfo();
+                if (!topology->getPropBool("@disableCachePrewarming", false))
```

Comment:
Not directly related but seems worth having an option for assisting in diagnosis of future issues in this area